plasmapy/classes/simulation.py

```diff
@@ -45,7 +46,7 @@ class MHDSimulation():
     def __init__(self, plasma):
         """
         """
-        self.dt = 0.01 * u.s
+        self.dt = 1e-4 * u.s
```

Comment:
Yeah, at this point it's just a value that makes it run. Ideally I'd like to have initial dt set by the user, with the option of using an adaptive timestep.